doc/Caliptra.md

```diff
@@ -233,116 +233,116 @@ In accordance with OCP Attestation specification [Reference 8](#ref-8), devices
-| Electric – Active     | Attacker alters the electrical signal or characteristics of external pins. | <p>Includes both analog attacks and digital signal attacks.</p><p>Excludes attacks requiring removing the package lid.</p> |
-| Temperature – Passive | Attacker observes the temperature of the product or portions of the product.| Excludes attacks requiring removing the package lid. |
-| Temperature – Active  | Attacker applies external heat sources or sinks to alter the temperature of the product, possibly in a rapid fashion. | <p>Includes all temperature ranges (e.g. pouring liquid nitrogen over the package or heating the package to above 100C)</p><p>Excludes attacks requiring removing the package lid.</p>|
-| Sound - Passive | Attacker observes the sounds emitted by the product. | <p>Includes all frequencies.</p><p>Excludes attacks requiring removing the package lid.</p> |
+| Electromagnetic – passive | Attacker observes the electromagnetic power spectrum and signals radiated from the product. | <p>Includes all attacks at all frequency ranges, including radio frequencies, infrared, optical, and ultraviolet.</p><p>Excludes attacks that require removing the package lid.</p> |
+| Electromagnetic – active  | Attacker directs electromagnetic radiation at the product or portions of the product. | <p>Includes all attacks at all frequency ranges, including radio frequencies, infrared, optical, and ultraviolet.</p><p>Excludes attacks that require removing the package lid.</p> |
+| Electric – passive    | Attacker probes the external pins of the package and observes electrical signals and characteristics including capacitance, current, and voltage signal. | <p>Includes both analog attacks and digital signal attacks.</p><p>Excludes attacks that require removing the package lid.</p> |
+| Electric – active     | Attacker alters the electrical signal or characteristics of external pins. | <p>Includes both analog attacks and digital signal attacks.</p><p>Excludes attacks that require removing the package lid.</p> |
+| Temperature – passive | Attacker observes the temperature of the product or portions of the product.| Excludes attacks that require removing the package lid. |
+| Temperature – active  | Attacker applies external heat sources or sinks to alter the temperature of the product, possibly in a rapid fashion. | <p>Includes all temperature ranges (for example, pouring liquid nitrogen over the package or heating the package to above 100C).</p><p>Excludes attacks that require removing the package lid.</p>|
+| Sound - passive | Attacker observes the sounds emitted by the product. | <p>Includes all frequencies.</p><p>Excludes attacks that require removing the package lid.</p> |
 
-*Table 5: Logical Attacks*
+*Table 5: Logical attacks*
 
-| **Attack** | **Description** | **Included / Excluded** |
+| **Attack** | **Description** | **Included and excluded** |
```

Comment:
"Included or excluded" ?

Reply:
most often, slashes denote and/or. Usually but not always "and" is best replacement. In this case, I'd actually recommend that we replace column header to be clearer and parallel with table 4: Threat model scope

Reply:
ack